peps/pep-0795.rst

```diff
@@ -2,7 +2,7 @@ PEP: 795
 Title: Deep Immutability for Efficient Sharing and Concurrency Safety
 Author: Matthew Johnson <matjoh@microsoft.com>, Matthew Parkinson <mattpark@microsoft.com>, Sylvan Clebsch <sylvan.clebsch@microsoft.com>, Fridtjof Stoldt <fridtjof.stoldt@it.uu.se>, Tobias Wrigstad <tobias.wrigstad@it.uu.se>
 Sponsor: Michael Droettboom <mdboom@gmail.com>
-Discussions-To: Pending
+Discussions-To: https://discuss.python.org/t/pep-795-revamped-deep-immutability-for-efficient-sharing-and-concurrency-safety/105356
```

Comment:
Please also include this in a `Post-History` field right after `Python-Version`, along with the first topic:

```rst
Post-History:
  `19-Jun-2025 <https://discuss.python.org/t/pep-795-add-deep-immutability-to-python/96014`__,
  `19-Dec-2025 <https://discuss.python.org/t/pep-795-revamped-deep-immutability-for-efficient-sharing-and-concurrency-safety/105356>`__,
```

And a reminder about the reminder, please wait for the PEP to be merged before opening the main discussion topic: https://github.com/python/peps/pull/4468#issuecomment-2987903203 :)

Reply:
My apologies and thanks! In my defense this is the same PEP? ;) I will try to remember this from now on. 

Also, will add the Post-History now. Thanks for all the help!